### ISSUE TITLE
3.0 - Add Order: JS error thrown when selecting an existing customer

### ISSUE THREAD

**Issue by cklosowski** [easydigitaldownloads/easy-digital-downloads#7068]
### Expected behavior
When selecting an existing customer while adding a new payment, no JS errors should be thrown.

### Actual behavior
The following error is being triggered in JS when selecting an existing customer from the Add Order screen:
```
TypeError: Cannot read property 'html' of null
 /wp-content/plugins/easy-digital-downloads/assets/js/edd-admin-orders.js?ver=1544466215:263
```

### Steps to reproduce the behavior
1) Orders > Add New
2) Click 'Assign' under the customer section, and pick a customer from the chosen dropdown.

### Information (if a specific version is affected):
PHP Version: All

EDD Version (or branch): `release/3.0`

WordPress Version: All

**Comment by spencerfinnell** [comment]
This seems to only happen to me when I select "No customer attached" again.

**Comment by spencerfinnell** [comment]
https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/includes/ajax-functions.php#L1155

This (and other instances) should probably be using `return wp_send_json_error()`

**Comment by cklosowski** [comment]
I think that this is a case of overlapping code. This customer address field loads in the other tab, and I do have an address, so I need to dig into why we got an error here or if this code was intended for another view.

**Comment by spencerfinnell** [comment]
@cklosowski https://github.com/easydigitaldownloads/easy-digital-downloads/pull/7069 fixes it for me.

When selecting a customer it tries to fetch that customer's address information. Currently if something goes wrong (like `customer_id` being 0 and then not finding a customer) the code will eventually just call `edd_die()` and break all the things. I opened another issue to fix this problem in other functions I noticed.
